src/utils/exportUtils/JSONExport.ts

```diff
@@ -108,7 +108,7 @@ export default class JSONExporter extends Exporter {
             this.addFile("export.json", new Blob([text]));
             await this.downloadZIP();
         } else {
-            const fileName = `matrix-export-${formatFullDateNoDay(new Date())}.json`;
+            const fileName = `${this.room.name}-${formatFullDateNoDayISO(new Date())}.json`;
```

Comment:
applies throughout: what if the room name contains characters that can't be translated into a file name?

Reply:
ohh i understand, i probably should check the room name first, if it contains characters that can't be used as a filename, then i default to the brand name or so, how about that?